Remotion/ObjectBinding/Web.ClientScript/tsconfig/release/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "outDir": "../../bin/Release/dist",
+    "outFile": "../../bin/Release/dist/ObjectBinding.Web.ClientScript.js",
```

Comment:
```suggestion
    "outFile": "../../bin/Release/dist/Remotion.ObjectBinding.Web.ClientScript.js",
```
Alternative: just "ClientScript.js" and the folder will take care of making the files distinct. The long name is perhaps easier to distinguish in the F12 tools since it's "Remotion.Obj..." vs "Remotion.Web..."

Reply:
I think the longer name is better exactly for the reason you mentioned. When searching in the dev tools it will be easier to distinguish the two scripts from each other.